foursight_core/react/api/gac.py

```diff
@@ -28,10 +28,11 @@ def get_secrets_names() -> list:
     @staticmethod
     def get_gac_names() -> list:
         secrets_names = Gac.get_secrets_names()
-        return [secret_name for secret_name in secrets_names if re.match('.*App(lication)?Config(uration)?.*', secret_name, re.IGNORECASE)]
+        pattern = ".*App(lication)?Config(uration)?.*"
+        return [secret_name for secret_name in secrets_names if re.match(pattern, secret_name, re.IGNORECASE)]
 
     @staticmethod
-    @memoize(100)
+    @lru_cache(100)
```

Comment:
See my comment elsewhere about `memoize`. If you do that, you could just import that definition here. But having this literal constant strewn about makes it hard to change this value in an organized way.

Reply:
Yeah I like it. Thanks. Updating.